modules/ocf_stats/manifests/grafana.pp

```diff
@@ -0,0 +1,62 @@
+# grafana config
+class ocf_stats::grafana {
+  class { 'ocf::packages::grafana::apt':
+    stage => first,
+  }
+
+  $canonical_url = $::host_env ? {
+    'dev'  => 'dev-prometheus.ocf.berkeley.edu',
+    'prod' => 'prometheus.ocf.berkeley.edu',
+  }
+
+  class { 'grafana':
+    install_method           => 'repo',
+
+    # Unfortunately, puppet-grafana requires that we specify a version number
+    # here.
+    version                  => '5.2.1',
+    manage_package_repo      => false,
+
+    cfg                      => {
+      app_mode         => 'production',
+      server           => {
+        http_addr => '127.0.0.1',
+        http_port => 8990,
+        domain    => $canonical_url,
+        root_url  => "https://${canonical_url}/grafana/",
+      },
+      database         => {
+        type => 'sqlite3',
+        path => 'grafana.db',
+      },
+      users            => {
+        auto_assign_org      => true,
+        allow_sign_up        => false,
+        auto_assign_org_role => 'Editor',
+      },
+      'auth.anonymous' => {
+        enabled  => true,
+      },
+      security         => {
+        disable_gravatar => true,
```

Comment:
Why? We use these for staff hours, so they should be reasonably useful if enabled

Reply:
Also, without an admin user here, who would have access to administer the grafana instance? Would that all be done through puppet? (That's fine if it's the case, I'm just not sure what your plan on this is)